server/src/main/java/org/elasticsearch/search/fetch/FetchSubPhase.java

```diff
@@ -45,6 +47,7 @@ public HitContext(
             LeafReaderContext context,
             int docId,
             SourceLookup sourceLookup,
+            CheckedBiConsumer<Integer, FieldsVisitor, IOException> fieldReader,
```

Comment:
Can you remove this change now that SourceLookup leverages the sequential reader automatically ?